.github/workflows/build-lint-test.yml

```
@@ -1,4 +1,4 @@
-name: Build, Lint, and Test
+name: Build, Lint, Test
```

Whoops. I didn't mean to delete the `and` :sweat_smile: 

```suggestion
name: Build, Lint, Test
```
```suggestion
name: Build, Lint, and Test
```